unstructured/ingest/connector/s3_connector.py

```diff
@@ -170,11 +188,16 @@ def fetch_docs(self):
         s3_keys = self._list_objects()
         return [
             S3IngestDoc(
-                self.config.s3_bucket,
+                self.config,
                 s3_key,
-                self.config.output_dir,
-                self._tmp_download_dir,
-                self.config.anonymous,
             )
+            # S3IngestDoc(
```

Comment:
Should this commented out section be here?

Reply:
no, removed. thanks for flagging.